h2/pom.xml

```diff
@@ -41,7 +41,7 @@
     <maven.compiler.target>1.7</maven.compiler.target>
     <derby.version>10.10.1.1</derby.version>
     <osgi.version>4.2.0</osgi.version>
-    <slf4j.version>1.6.0</slf4j.version>
+    <slf4j.version>1.7.25</slf4j.version>
```

Comment:
this is now different to both our manifest and the version we use in Build.java ?

Reply:
Urgh, I didn't even see that you have what appears to be two build systems. I've dropped this commit since it's clearly out of scope of this PR now.